### DISCUSSION TITLE
about custom keybinds

### DISCUSSION THREAD

**Discussion by samarth-na** [jorgerojas26/lazysql#135] [Q&A]
I can see that custom keybinds is in the roadmap. I want to contribute to it, can someone point me and likes of me to it so I, we can work on it 

To me its the only reason I cant use lazysql the default keybinds it has in the start screen `f1, f2, f3` for me are reserved to some OS shortcuts

**Comment by jorgerojas26** [comment]
Hi, if you want to contribute to the custom keybindings feel free to open a PR, that's a really good feature i want for lazysql, but it is not the first in my list (query history, for example, is more important for me right now). Having said that, you can alternatively save a new connection by pressing "Enter" in the form, that can replace the F1. For the F2, there is not an alternative for that, but i think that's not a stopper for using lazysql.

Also, i'm about to merge a PR where you can just use the command: "lazysql [connection_url]" and it will just connect to it.

I think it can be very straightforward to do the custom keybinding feature, there's already a config file, there's a central keybinding system as well, so i think it's just a matter of creating the keybindings in the config file, read and load it at start.

**Comment by samarth-na** [comment]
thanks for the information I'll try to make something work
